src/server/replica.h

```diff
@@ -234,6 +234,7 @@ class DflyShardReplica : public ProtocolClient {
   // **executed** records, which might be received interleaved when commands
   // run out-of-order on the master instance.
   std::atomic_uint64_t journal_rec_executed_ = 0;
+  std::atomic_uint64_t lsn_ = 0;
```

Comment:
why it is atomic?

Reply:
It shouldn't be, just like `journal_rec_executed` and some other atomics. I will patch this, give me a second